integration/src/main/java/scripts/testscripts/ControlledGcsBucketLifecycle.java

```diff
@@ -219,6 +219,20 @@ public void doUserJourney(TestUserSpecification testUser, WorkspaceApi workspace
         resource.getMetadata().getDescription());
     assertEquals(UPDATED_RESOURCE_NAME, resource.getMetadata().getName());
     assertEquals(UPDATED_DESCRIPTION, resource.getMetadata().getDescription());
+    // However, invalid updates are rejected.
+    String invalidName = "!!!invalid_name!!!";
+    ApiException invalidUpdateEx =
+        assertThrows(
+            ApiException.class,
+            () ->
+                updateBucketAttempt(
+                    resourceApi,
+                    resourceId,
+                    invalidName,
+                    /*updatedDescription=*/ null,
```

Comment:
IntelliJ shows me these parameter names when I pass a literal value, so I haven't felt the need to put these comments. Is it only the pro version that does this?

Reply:
Community edition does that too, but other editors probably don't (and it's nice when I'm reading the code on Github)